recipes/protopuf/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.47.0"
```
1.47.0 should be enough here